myapp/db/migrate/20220805081451_create_tasks.rb

```diff
@@ -0,0 +1,14 @@
+class CreateTasks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tasks do |t|
+      t.string :title, :limit => 128, :null => false, :default => ''
```

Comment:
ハッシュの書き方について、現場に合わせる形となってしまいますが、
以下のように`=>`を使わない書き方で統一していただきたいです。
```suggestion
      t.string :title, limit: 128, null: false, default: ''
```

Reply:
> レビューしましたのでご確認お願いいたします。
またPRの説明欄について、次回から以下の項目を追記していただけると幸いです。
> 
> 要件
> 各STEPに記載されている課題をそのまま転記で問題ありません
> 動作確認（スクリーンショット等）
> 実装したものを確認する手順を記載してください
> 画面を作成・修正した場合は画面のスクリーンショットを、テーブルを作成・修正した場合は変更後のテーブル構成がわ> かるスクリーンショットを添付してください
追記しました。

> ハッシュの書き方について、現場に合わせる形となってしまいますが、
> 以下のように=>を使わない書き方で統一していただきたいです。
ご指摘のように```=>```を使用しない記述に修正しました。